### ISSUE TITLE
title is empty and album is missing from Metadata

### ISSUE THREAD

**Issue by ChrisOboe** [jecaro/mprisqueeze#3]
The Metadata property has an empty title and no album at all. The artist is there. 

Here a screenshot from D-Spy
![grafik](https://github.com/user-attachments/assets/d76dc2e6-38af-4d11-ae7f-d22ca328e86d)

Here a screenshot from a different MPRIS provider (kdeconnect, grabbing the data from squeezer on my phone) as comparison
![grafik](https://github.com/user-attachments/assets/76c88257-80e8-4703-b154-7eab7d0dc8ae)

**Comment by jecaro** [comment]
Thanks for reporting this issue.

Yeah. Album is not currently implemented (I have no use of it) but it should be pretty easy to add it. I'll definitely try to find time to implement it soon. Otherwise if you're willing to do it yourself feel free to send a PR.

As for the title missing, I'm interested in digging a bit further to understand what is going on. Could you run `mprisqueeze` with debug logging on?

```bash
$ RUST_LOG=debug mprisqueeze -H somehost -P 9000  
```

It'll output the requests sent to the server along its responses. For example here is what I have:

```
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("shinystuff", ["mode", "?"]) }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("shinystuff", ["mode", "?"]), result: Object {"_mode": String("pause")} }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Converted as: Pause
[2024-09-22T19:07:08Z DEBUG mprisqueeze::mpris] MprisPlayer::loop_status
[2024-09-22T19:07:08Z DEBUG mprisqueeze::mpris] MprisPlayer::shuffle
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("shinystuff", ["playlist", "shuffle", "?"]) }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("shinystuff", ["playlist", "shuffle", "?"]), result: Object {"_shuffle": Number(0)} }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Converted as: Off
[2024-09-22T19:07:08Z DEBUG mprisqueeze::mpris] MprisPlayer::metadata
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("shinystuff", ["playlist", "tracks", "?"]) }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("shinystuff", ["playlist", "tracks", "?"]), result: Object {"_tracks": Number(9)} }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Converted as: 9
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("shinystuff", ["current_title", "?"]) }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("shinystuff", ["current_title", "?"]), result: Object {"_current_title": String("The Beginning and the End")} }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Converted as: Some("The Beginning and the End")
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("shinystuff", ["artist", "?"]) }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("shinystuff", ["artist", "?"]), result: Object {"_artist": String("Isis")} }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Converted as: Some("Isis")
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("shinystuff", ["playlist", "index", "?"]) }
[2024-09-22T19:07:08Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("shinystuff", ["playlist", "index", "?"]), result: Object {"_index": String("0")} }
```

Also which version of LMS are you running?

**Comment by jecaro** [comment]
I've been running the version from [this branch](https://github.com/jecaro/mprisqueeze/tree/album-title-fix) this week. It contains support for the album, and I've used another call to LMS for the title.

```bash
$ playerctl metadata
shinystuff mpris:trackid             '/org/mpris/MediaPlayer2/shinystuff/track/10'
shinystuff xesam:album               Perfect From Now On
shinystuff xesam:artist              Built to Spill
shinystuff xesam:title               Untrustable / Part 2 (About Someone Else)

```

Let me know if it fixes your issue. I'll merge it sometime next week. I want to keep using it for a few days, to make sure it runs fine.

**Comment by ChrisOboe** [comment]
Sorry for the late reply and thank you for your fast fix. 
My lms servier is lyron 8.5.2
With the album-title-fix branch it works as expected. 

<details>
<summary>my debug log of the main branch where the title and album was missing</summary>

```
2024-10-03T06:05:26Z DEBUG mprisqueeze] Options: Options { hostname: None, port: 9000, player_name: "jeff", player_timeout: 3, discover_timeout: 3, discover_reply_timeout: 100, player_command: ["/nix/store/hvykmsj63clzpb310747q0d0djsmrkzg-squeezelite-pulse-2.0.0.1488/bin/squeezelite-pulse", "-n", "{name}", "-s", "{server}"] }
[2024-10-03T06:05:26Z INFO  mprisqueeze::discover] Discovering LMS server on the local network
[2024-10-03T06:05:26Z INFO  mprisqueeze::discover] Found LMS server: chump:9000 (8.5.2)
Discovered LMS at chump:9000
[2024-10-03T06:05:26Z INFO  mprisqueeze] Starting player: /nix/store/hvykmsj63clzpb310747q0d0djsmrkzg-squeezelite-pulse-2.0.0.1488/bin/squeezelite-pulse ["-n", "jeff", "-s", "chump"]
[2024-10-03T06:05:26Z INFO  mprisqueeze] Waiting for player jeff to be available
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("", ["player", "count", "?"]) }
[2024-10-03T06:05:26Z DEBUG reqwest::connect] starting new connection: http://chump:9000/
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("", ["player", "count", "?"]), result: Object {"_count": Number(4)} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 4
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("", ["players", "0"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("", ["players", "0"]), result: Object {"count": Number(4), "players_loop": Array [Object {"canpoweroff": Number(1), "connected": Number(1), "displaytype": String("none"), "firmware": String("v2.0.0-1488"), "ip": String("192.168.24.217:44444"), "isplayer": Number(1), "isplaying": Number(0), "model": String("squeezelite"), "modelname": String("SqueezeLite"), "name": String("fulbert"), "playerid": String("00:00:00:00:00:00"), "playerindex": String("0"), "power": Number(0), "seq_no": Number(0), "uuid": Null}, Object {"canpoweroff": Number(1), "connected": Number(0), "displaytype": String("none"), "firmware": String("v2.0.0-1488"), "ip": String("192.168.24.169:41956"), "isplayer": Number(1), "isplaying": Number(1), "model": String("squeezelite"), "modelname": String("SqueezeLite"), "name": String("jeff"), "playerid": String("b4:2e:99:f5:e7:50"), "playerindex": Number(1), "power": Number(1), "seq_no": Number(0), "uuid": Null}, Object {"canpoweroff": Number(1), "connected": Number(0), "displaytype": String("none"), "firmware": String("v2.0.0-1488"), "ip": String("192.168.24.204:49138"), "isplayer": Number(1), "isplaying": Number(0), "model": String("squeezelite"), "modelname": String("SqueezeLite"), "name": String("werblount"), "playerid": String("a8:64:f1:2a:f9:8b"), "playerindex": Number(2), "power": Number(1), "seq_no": Number(0), "uuid": Null}, Object {"canpoweroff": Number(1), "connected": Number(1), "displaytype": String("none"), "firmware": String("v1.0-1670-16"), "ip": String("192.168.24.147:61836"), "isplayer": Number(1), "isplaying": Number(0), "model": String("squeezeesp32"), "modelname": String("SqueezeESP32"), "name": String("buero"), "playerid": String("08:b6:1f:b7:ad:7c"), "playerindex": Number(3), "power": Number(1), "seq_no": Number(0), "uuid": Null}]} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: [Player { name: "fulbert" }, Player { name: "jeff" }, Player { name: "werblount" }, Player { name: "buero" }]
[2024-10-03T06:05:26Z INFO  mprisqueeze] Player jeff is available
[2024-10-03T06:05:26Z INFO  mprisqueeze::mpris] Starting DBus server for player jeff
[2024-10-03T06:05:26Z INFO  mprisqueeze::mpris] DBus server started for player jeff
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::identity
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::playback_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["mode", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::can_quit
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::can_raise
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::has_track_list
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::identity
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::supported_uri_schemes
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::supported_mime_types
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::playback_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["mode", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::can_quit
[2024-10-03T06:05:26Z DEBUG reqwest::connect] starting new connection: http://chump:9000/
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::can_raise
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::has_track_list
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::identity
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::supported_uri_schemes
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisRoot::supported_mime_types
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["mode", "?"]), result: Object {"_mode": String("play")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Play
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::loop_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::shuffle
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["mode", "?"]), result: Object {"_mode": String("play")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Play
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::loop_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::shuffle
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]), result: Object {"_shuffle": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Songs
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::metadata
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "tracks", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]), result: Object {"_shuffle": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Songs
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::metadata
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "tracks", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "tracks", "?"]), result: Object {"_tracks": Number(27)} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 27
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["current_title", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["current_title", "?"]), result: Object {"_current_title": String("")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Some("")
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["artist", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "tracks", "?"]), result: Object {"_tracks": Number(27)} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 27
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["current_title", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::metadata
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "tracks", "?"]) }
[2024-10-03T06:05:26Z DEBUG reqwest::connect] starting new connection: http://chump:9000/
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["current_title", "?"]), result: Object {"_current_title": String("")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Some("")
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["artist", "?"]), result: Object {"_artist": String("Ice Nine Kills")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["artist", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Some("Ice Nine Kills")
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "index", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "tracks", "?"]), result: Object {"_tracks": Number(27)} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 27
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["current_title", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "index", "?"]), result: Object {"_index": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 1
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::volume
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::position
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::minimum_rate
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::maximum_rate
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_go_next
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_go_previous
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_play
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_pause
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_seek
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_control
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["artist", "?"]), result: Object {"_artist": String("Ice Nine Kills")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Some("Ice Nine Kills")
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "index", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::position
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["current_title", "?"]), result: Object {"_current_title": String("")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Some("")
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["artist", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "index", "?"]), result: Object {"_index": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 1
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::volume
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::position
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::minimum_rate
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::maximum_rate
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_go_next
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_go_previous
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_play
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_pause
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_seek
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_control
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::position
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["artist", "?"]), result: Object {"_artist": String("Ice Nine Kills")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Some("Ice Nine Kills")
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "index", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "index", "?"]), result: Object {"_index": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: 1
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::position
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::playback_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["mode", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["mode", "?"]), result: Object {"_mode": String("play")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Play
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_pause
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_play
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_go_next
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_go_previous
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::can_seek
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::loop_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::loop_status
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::shuffle
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]), result: Object {"_shuffle": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Songs
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::shuffle
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Sending: LmsRequest { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]) }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Received: LmsResponse { method: "slim.request", params: ("jeff", ["playlist", "shuffle", "?"]), result: Object {"_shuffle": String("1")} }
[2024-10-03T06:05:26Z DEBUG mprisqueeze::lms] Converted as: Songs
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::volume
[2024-10-03T06:05:26Z DEBUG mprisqueeze::mpris] MprisPlayer::position
```
</details>

**Comment by jecaro** [comment]
Thx for your feedback.

Looking at the debug log, we can see that the `current_title` request returns an empty string in your case. I can't remember why I chose `current_title` over `title` when implementing this :thinking: 

Anyway, I've merged this branch and created a new release. 

Closing as it is now fixed in [v0.1.7](https://github.com/jecaro/mprisqueeze/releases/tag/v0.1.7)
